app/javascript/src/ActiveDocs/customize/plugins.js

```diff
@@ -0,0 +1,4 @@
+import { WrappedComponentsPlugin } from 'ActiveDocs/customize/wrappedComponents'
```

Comment:
the path `customize` is named after a nomenclature convention from swagger? if not, something like `custom` might do.